modules/administration/pages/auth-methods.adoc

```diff
@@ -2,99 +2,102 @@
 = Authentication Methods
 
 
+
 == Authentication Via PAM
 
 
-As security measures become increasingly complex, SUSE Manager supports network-based authentication systems via Pluggable Authentication Modules (PAM). PAM is a suite of libraries that allows to integrate SUSE Manager with a centralized authentication mechanism, thus eliminating the need to remember multiple passwords.
-SUSE Manager supports LDAP, Kerberos, and other network-based authentication systems via PAM.
-To enable SUSE Manager to use PAM in your organization's authentication infrastructure, follow the steps below.
+As security measures become increasingly complex, {productname} supports network-based authentication systems via Pluggable Authentication Modules (PAM).
+PAM is a suite of libraries that allows to integrate {productname} with a centralized authentication mechanism, thus eliminating the need to remember multiple passwords.
```

Comment:
```suggestion
PAM is a suite of libraries that allows you to integrate {productname} with a centralized authentication mechanism, eliminating the need to remember multiple passwords.
```

Reply:
Do people use PAM so that they don't need to remember passwords? Isn't it more about connecting to a workplace user db?